CHANGELOG.md

```diff
@@ -59,6 +59,15 @@ The following utils have had minor internal changes to satisfy the introduction
 * Router
 * Service
 
+The following utils have had minor internal changes to satisfy the introduction of stricter linting rules:
+
+* SerializeHelper
+* ShouldComponentUpdateHelper
+* TagsHelper
+* ToArrayHelper
+* ValidationsHelper
```

Comment:
Do we need to list all the helpers, or can we just write 'Helpers'?
(see https://github.com/Sage/carbon/pull/1338/files?short_path=4ac32a7#diff-4ac32a78649ca5bdd8e0ba38b7006a1e - this matches how we've listed Validations without listing each individual validation).